docs/Intersections/README.md

```diff
@@ -0,0 +1,25 @@
+# Intersection API Documentation
+
+This directory contains documentation about the Intersection api. This API was created as a fork of the [CIMMS ConflictVisualizer Api](https://github.com/usdot-jpo-ode/jpo-conflictvisualizer/tree/cvmgr-cimms-integration/api).
+
+All of the data flowing into and out of the intersection api is shown in the following dataflow diagram:
+![Intersection Api Dataflow Diagram](../../docs/Intersections/IntersectionApiDataFlows.png)
```

Comment:
suggestion(blocking): this link doesn't seem to work due to the actual filename having spaces (at least in my vscode markdown preview extension)